src/NuGet.Services.Search/NuGet.Services.Search.csproj

```diff
@@ -188,7 +188,7 @@
       <HintPath>..\..\packages\NuGet.Core.2.8.5\lib\net40-Client\NuGet.Core.dll</HintPath>
     </Reference>
     <Reference Include="NuGet.Indexing, Version=0.0.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\NuGet.Indexing.3.0.280-r-master\lib\net45\NuGet.Indexing.dll</HintPath>
```

Comment:
I'm making use of a newly added event exposed on SearcherManager in this PR. See https://github.com/NuGet/NuGet.Services.Search/pull/32/files#diff-7e25d8226c9211da5c04ab4be5c88a22R44

Either we wait for the consolidation, or the v2/v3 merge stuff should be branched off (into a 'dev' branch?) and reverted on this branch. Any preference? :)